---
title: 컴파일러 오류 CS1631
ms.date: 07/20/2015
f1_keywords:
- CS1631
helpviewer_keywords:
- CS1631
ms.assetid: bf0c5ff9-90a3-4db6-b4ee-0b93e31614e0
ms.openlocfilehash: 7dc7473966b1c78b911c1323b5ea0f8fbf7d8150
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 04/23/2019
ms.locfileid: "61653133"
---
# <a name="compiler-error-cs1631"></a>컴파일러 오류 CS1631
catch 절 본문에서는 값을 생성할 수 없습니다.  
  
 yield 문은 catch 절의 본문 내에서 허용되지 않습니다. 이 오류를 방지하려면 yield 문을 catch 절 본문 밖으로 이동합니다.  
  
 다음 샘플에서는 CS1631을 생성합니다.  
  
```csharp  
// CS1631.cs  
using System;  
using System.Collections;  
  
public class C : IEnumerable  
{  
   public IEnumerator GetEnumerator()   
   {  
      try  
      {  
      }  
      catch(Exception e)  
      {  
        yield return this;  // CS1631  
      }  
   }    
  
   public static void Main()   
   {  
   }  
}  
```
